app.py

```diff
@@ -339,7 +356,7 @@ async def command(ack, body, respond, client, logger):
                 "type": "plain_text_input",
                 "multiline": True,
                 "action_id": "plain_text_input-action",
-                "initial_value": "WARMUP: \nTHE THANG: \nMARY: \nANNOUNCEMENTS: \nCOT: ",
+                "initial_value": "Warm-Up: \nThe Thang: \nMary: \nAnnouncements: \nCOT: \nNaked-Man Moleskine: ",
```

Comment:
@kodesmith, adding the Conditions and Naked-Man Moleskin may be what many regions do, but I think across the board many would find it superfluous and adding clutter. I think this may be too opinionated. I'd be interested to hear what @wolfpackt99 or @nickspiers think.

Reply:
I like the start case capitalization change. It provides for better consistency with Title and Count, etc.

Reply:
I think if the just the docker improvements were put into one pull request and separate from these code changes, it could be approved more easily, @kodesmith.

Reply:
Thanks for the feedback.  The conditions came from PaxMiner (https://f3stlouis.com/paxminer/) but the moleskine was just from the nation (https://f3nation.com/lexicon/entry/2586/?gvid=398600)

I completely understand what you are saying though.  I will pull them out and create a PR with just the cleanup changes, one with the docker and readme changes, and call it a success.